frontend/src/index.css

```
@@ -142,6 +142,29 @@ body {
+  background-color: #fafaf9;
+  border: 1px solid #e7e5e4;
+  border-radius: 6px;
+  overflow: auto;
+  text-align: center;
+}
+.markdown-body .mermaid svg {
+  max-width: 100%;
+  height: auto;
+}
+.markdown-body .mermaid:not([data-processed]) {
+  font-family: 'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', monospace;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Monaco" (font-family-name-quotes)

(font-family-name-quotes)